helpers/errorlist/errorlist.go

```diff
@@ -0,0 +1,41 @@
+package errorlist
+
+import (
+	"fmt"
+)
+
+type ErrorList []error
+
+func (e ErrorList) Error() string {
+	s := ""
+
+	for idx, err := range e {
+		s = fmt.Sprintf("%d: %s\n", idx, err)
+	}
+
+	return s
+}
+
+// Append will append an error into [ErrorList]
+func (e ErrorList) Append(err error) ErrorList {
+	return append(e, err)
+}
```

Comment:
Why this package when errors.Join is available?

Reply:
ill check it, or maybe ill use []error since the errors is logged, not returned